### PR TITLE
[W5.4][T16-A1] Poon Wenzhe

### PR DESCRIPTION
**Pull request by pwenzhe**
Add ability to change themes.

### CODE REVIEW

File: src/seedu/addressbook/ui/MainWindow.java [hunk 1]
```diff
@@ -22,6 +24,7 @@
 
     private Logic logic;
     private Stoppable mainApp;
+    private int theme = 0;
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/ui/MainWindow.java:27]: Do you think that using of enum would improve the code quality?

File: src/seedu/addressbook/ui/MainWindow.java [hunk 2]
```diff
@@ -50,6 +53,9 @@ void onCommand(ActionEvent event) {
                 exitApp();
                 return;
             }
+            else if(isThemeCommand(result)) {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/ui/MainWindow.java:56]: [Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. missing spaces after `if`.

**Review by brandonyeoxg — COMMENTED** [review]
@pwenzhe Some comments added. Please close the PR after reading comments. You have fulfilled the requirements for the LO =).

File: src/seedu/addressbook/ui/MainWindow.java (hunk 1 above)
Comment (pwenzhe) [reply src/seedu/addressbook/ui/MainWindow.java:27]: Yes it would make the code more friendly, especially without the use of comments. Thank you.

**Comment by pwenzhe** [comment]
Read, and thank you for the comments.

**Pull request closed without merging**